cache/refs.go

```diff
@@ -1039,7 +1048,7 @@ func (sr *immutableRef) unlazyLayer(ctx context.Context, dhs DescHandlers, s ses
 		})
 	}
 
-	desc, err := sr.ociDesc(ctx, dhs)
+	desc, err := sr.ociDesc(ctx, dhs, true)
```

Comment:
Shouldn't this be false? Why drop the URL here?

Reply:
This is fixed due to `ociDesc` changing tense here.